server/routes/__testutils/appSetup.ts

```diff
@@ -24,19 +24,9 @@ export const user = {
 }
 
 export const signedCookiesProvider = jest.fn()
+export const flashProvider = jest.fn()
```

Comment:
I've made one instance to flash provider here so can just import one into the test where needed, don't need to create one per test and inject it in

Want to do the same with roles too.